### PR TITLE
Rust 1.26.2 announcement

### PR DESCRIPTION
**Pull request by Mark-Simulacrum**
This announcement is on the short side, but this release is also quite small. Should we talk further about why we're issuing a .2 release? I feel like the single sentence I included should be sufficient...

r? @rust-lang/core

### CODE REVIEW

File: _posts/2018-06-05-Rust-1.26.2.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.26.2][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/stable/RELEASES.md#version-1262-2018-06-05
+
+## What's in 1.26.2 stable
+
+An unsoundness in the borrow checker verification of match statements was found.
```
Comment (aturon) [inline _posts/2018-06-05-Rust-1.26.2.md]: I think we should call out that this is a bug in the just-stabilized changes to match that first appeared in 1.26.

Comment (Mark-Simulacrum) [reply _posts/2018-06-05-Rust-1.26.2.md]: How does that sound?

File: _posts/2018-06-05-Rust-1.26.2.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.26.2][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/stable/RELEASES.md#version-1262-2018-06-05
+
+## What's in 1.26.2 stable
+
+Unsound behavior was discovered in the borrow checker verification of match statements which use the
```
[the remark this answers is not in the document]
Comment (Mark-Simulacrum) [reply _posts/2018-06-05-Rust-1.26.2.md]: Updated the code a bit; what do you think?

[the remark this answers is not in the document]
Comment (tatsuya6502) [reply _posts/2018-06-05-Rust-1.26.2.md]: It looks good. Thank you!

**Comment by aturon** [comment]
@Mark-Simulacrum :+1: on the changes, thank you!

**Pull request merged**